### PR TITLE
Added EmailIt and Modal File Manager

### PR DESCRIPTION
**Pull request by raguay**
I think I added the two properly. Let me know if I need to change anything. Thanks.

### CODE REVIEW

**Comment by leaanthony** [comment]
Thanks for this 🙏 Did you intend to add the images too?

**Comment by raguay** [comment]
I believe I added the images to the correct place. If they need to be
different or moved around, please let me know.

File: website/docs/community/showcase/emailit.mdx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# EmailIt
+
+<p style={{"text-align": "center"}}>
+   <img src="/img/showcase/emailit.png"></img><br/>
```
Comment (leaanthony) [inline website/docs/community/showcase/emailit.mdx:4]: This Image does not exist in the repo or this PR

**Review by leaanthony — CHANGES_REQUESTED** [review]
I'd love to merge this but the lack of images will create broken pages. If you push the images to your branch this PR should pick them up 🙏

**Pull request closed without merging**